code/game/objects/items/food/lizard.dm

```diff
@@ -223,7 +223,7 @@
 	icon_state = "surf_n_turf"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 15, /datum/reagent/consumable/nutriment/protein = 10, /datum/reagent/consumable/nutriment/vitamin = 10)
 	tastes = list("surf" = 1, "turf" = 1)
-	foodtypes = MEAT | VEGETABLES
```

Comment:
The surf'n'turf has both meat and fish in it, keep the MEAT foodtype on this one.

Reply:
Fixed this, thank you